includes/html/collectd/functions.php

```diff
@@ -18,7 +18,7 @@
 
 require 'includes/html/collectd/CollectdColor.php';
 
-use LibreNMS\CollectdColor;
+use html\html\collectd\CollectdColor;
```

Comment:
Nah?

Reply:
Stupid IDE, I missed that one to revert.